packages/cdk-assets/lib/private/handlers/external-files.ts

```diff
@@ -0,0 +1,18 @@
+import { FileManifestEntry } from '../../asset-manifest';
+import { IAssetHandler, IHandlerHost } from '../asset-handler';
+import { shell } from '../shell';
+import { FileAssetHandler } from './files';
+
+export class ExternalFileAssetHandler extends FileAssetHandler implements IAssetHandler {
+  constructor(
+    protected readonly asset: FileManifestEntry,
+    protected readonly host: IHandlerHost) {
+    super('', asset, host);
+  }
+
+  protected async packageFile(): Promise<string> {
+    const source = this.asset.externalSource!;
+    const file = await shell(source.executable.split(' '));
```

Comment:
Where is `externalSource.packaging` used? 

* If it isn't, get rid of it.
* If it is, because we're aliasing `genericSource = source ?? externalSource` somewhere and the base class still uses that field, then I think that's a perfect example of awkward coupling that I'd like to avoid.

Reply:
Already removed in next fixup 😄